src/edge/tests/test_importer.py

```diff
@@ -548,4 +553,50 @@ def test_import_gff_CDS_subfragments_overlap_check_chunks(self, cf_fcl_mock):
         self.assertEquals(gene_chunk_starts, [30, 41, 42, 51, 56, 60, 62])
         self.assertEquals(gene_chunk_ends, [40, 41, 50, 55, 59, 61, 80])
         self.assertEquals(cds_chunk_starts, [30, 41, 41, 42, 56, 60, 60, 62])
-        self.assertEquals(cds_chunk_ends, [40, 41, 41, 50, 59, 61, 61, 80])
\ No newline at end of file
+        self.assertEquals(cds_chunk_ends, [40, 41, 41, 50, 59, 61, 61, 80])
+
+    def test_import_gff_CDS_subfragments_SGD_CDS(self):
+        data = """##gff-version 3
+chrI\tSGD\tchromosome\t1\t160\t.\t.\t.\tID=i1;Name=f1
+chrI\tSGD\tgene\t20\t60\t.\t+\t.\tID=A1;Name=A1;gene=gene_A1
+chrI\tSGD\tCDS\t20\t37\t.\t+\t0\tParent=A1_mRNA;Name=A1_CDS;orf_classification=Verified
+chrI\tSGD\tintron\t38\t39\t.\t+\t.\tParent=A1_mRNA;Name=A1_intron;orf_classification=Verified
+chrI\tSGD\tCDS\t40\t60\t.\t+\t0\tParent=A1_mRNA;Name=A1_CDS;orf_classification=Verified
+chrI\tSGD\tmRNA\t20\t60\t.\t+\t.\tID=A1_mRNA;Name=A1_mRNA;Parent=A1
+###
+##FASTA
+>chrI
+CCACACCACACCCACACACCCACACACCACACCACACACCACACCACACCCACACACACACATCCTAACACTACCCTAAC
+ACAGCCCTAATCTAACCCTGGCCAACCTGTCTCTCAACTTACCCTCCATTACCCTGCCTCCACTCGTTACCCTGTCCCAT
+"""
+
+        with tempfile.NamedTemporaryFile(mode="w+", delete=False) as f:
+            f.write(data)
+            f.close()
+            genome = Genome.import_gff("Foo", f.name)
+            os.unlink(f.name)
+
+        # created one fragment for each sequence in GFF file
+        self.assertCountEqual(
+            [fr.name for fr in genome.fragments.all()], ["chrI"]
+        )
+        chrI = [
+            fr.indexed_fragment() for fr in genome.fragments.all() if fr.name == "chrI"
+        ][0]
+        self.assertEquals(len(chrI.sequence), 160)
+        self.assertEquals(len(chrI.annotations()), 5)
+        self.assertEquals(chrI.annotations()[0].base_first, 20)
+        self.assertEquals(chrI.annotations()[0].base_last, 60)
+        self.assertEquals(chrI.annotations()[0].feature.name, 'A1')
```

Comment:
Can you check for each annotation's feature object, feature.type is correct?

Reply:
Can you ALSO check here, and in all the other tests, that the annotations belonging to the same feature not only has the same feature.name, but also feature.id (i.e. actually same rows in DB). Thanks.